packages/react-composites/src/composites/localization/locales/it-IT/strings.json

```diff
@@ -396,6 +404,8 @@
     "dialpadModalAriaLabel": "Tastiera",
     "dialpadCloseModalButtonAriaLabel": "Chiudi tastiera",
     "openDtmfDialpadLabel": "Mostra tastiera",
-    "dtmfDialpadPlaceholderText": "Immetti il numero"
+    "dtmfDialpadPlaceholderText": "Immetti il numero",
+    "breakoutRoomChatPaneTitle": "Chat della stanza di lavoro",
+    "chatContentSpinnerLabel": "Loading..."

```

Comment:
We will need to pull in the strings again later this week because "Loading..." was merged in main late last week.